IdentityCore/src/broker_operation/response/MSIDBrokerOperationAccountResponse.m

```diff
@@ -0,0 +1,65 @@
+//
+//  MSIDBrokerOperationAccountResponse.m
+//  IdentityCore iOS
+//
+//  Created by JZ on 10/1/19.
+//  Copyright © 2019 Microsoft. All rights reserved.
+//
+
+#import "MSIDBrokerOperationAccountResponse.h"
+#import "MSIDAccount+MSIDJsonSerializable.h"
+
+@implementation MSIDBrokerOperationAccountResponse
+
+#pragma mark - MSIDJsonSerializable
+
+- (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)error
+{
+    self = [super initWithJSONDictionary:json error:error];
+    
+    if (self)
+    {
+        if (![json msidAssertType:NSDictionary.class
+                          ofField:@"response_data"
+                          context:nil
+                        errorCode:MSIDErrorInvalidInternalParameter
+                            error:error])
+        {
+            return nil;
+        }
+        
+        NSMutableArray *accounts = [NSMutableArray new];
+        
+        NSArray *accountsJson = json[@"response_data"][@"accounts"];
```

Comment:
Should we validate that accountsJson is Array of Dictionaries?

Reply:
+1,  we should. Also validate the response_data is a dictionary.

Reply:
addressed :)